wpilib/wpilib/sendablebase.py

```diff
@@ -0,0 +1,96 @@
+import threading
+from .sendable import Sendable
+from .livewindow import LiveWindow
+from ._impl.utils import match_arglist
+
+
```

Comment:
Define `__all__`, otherwise the `import *` will pick up everything that's been imported.